cloudapi/api.go

```diff
@@ -258,3 +265,27 @@ func (c *Client) Login(email string, password string) (*LoginResponse, error) {
 
 	return &lr, nil
 }
+
+// ValidateToken calls the endpoint to validate the Client's token and returns the result.
+func (c *Client) ValidateToken() (*ValidateTokenResponse, error) {
+	url := fmt.Sprintf("%s/validate-token", c.baseURL)
```

Comment:
nit: this can be a string concatination 
```suggestion
	url := c.baseURL + "/validate-token"
```

I am pretty sure a bunch of the other such operations here can be concatinations as well.